src/vaccel_args.c

```diff
@@ -0,0 +1,133 @@
+
+#include "vaccel_args.h"
+#include <stdio.h>
+#include <string.h>
+#include <stdlib.h>
+
+struct vaccel_arg_list* vaccel_args_init(){
+
+    struct vaccel_arg_list* arg_list = 
+        (struct vaccel_arg_list*)malloc(sizeof(struct vaccel_arg_list));
+
+    arg_list->size = 0;
+    arg_list->list = (struct vaccel_arg*)malloc(0);
```

Comment:
We do that because we later call realloc() on arg_list->list. Realloc() requires that the memory area that is about to modify, has been previously allocated with malloc or equivalent